tests/admin/test_forms.py

```diff
@@ -19,29 +20,51 @@
 from enterprise.admin.forms import (EnterpriseCustomerAdminForm, EnterpriseCustomerIdentityProviderAdminForm,
                                     ManageLearnersForm)
 from enterprise.admin.utils import ValidationMessages
-from enterprise.course_catalog_api import NotConnectedToOpenEdX, get_all_catalogs
-from test_utils import fake_enrollment_api
+from enterprise.course_catalog_api import CourseCatalogApiClient
+from test_utils import fake_catalog_api, fake_enrollment_api
 from test_utils.factories import (EnterpriseCustomerFactory, EnterpriseCustomerUserFactory,
                                   PendingEnterpriseCustomerUserFactory, SiteFactory, UserFactory)
+from test_utils.fake_catalog_api import FAKE_PROGRAM_RESPONSE1, FAKE_PROGRAM_RESPONSE2
 
 FAKER = FakerFactory.create()
 
 
+class TestWithCourseCatalogApiMixin(object):
+    """
+    Mixin class to provide CourseCatalogApiClient mock.
+    """
+    def _make_catalog_api(self):
+        """
+        Build CourseCatalogApiClient mock and register it for deactivation at cleanup.
+        :return:
+        """
+        patched_api = mock.Mock(CourseCatalogApiClient)
+        patcher = mock.patch('enterprise.admin.forms.CourseCatalogApiClient', mock.Mock(return_value=patched_api))
+        patcher.start()
+        self.addCleanup(patcher.stop)
+        return patched_api
+
+
 @mark.django_db
 @ddt.ddt
-class TestManageLearnersForm(unittest.TestCase):
+class TestManageLearnersForm(unittest.TestCase, TestWithCourseCatalogApiMixin):
```

Comment:
Let's place `TestWithCourseCatalogApiMixin` before `unittest.TestCase` here. That way, its `setUp` method will be called.